### ISSUE TITLE
B301 gives a false positive for pandas DataFrames, Series and Panels

### ISSUE THREAD

**Issue by mxr** [PyCQA/flake8-bugbear#42]
`iteritems` is a valid function for pandas DataFrames, Series, and Panels (see docs [here](http://pandas.pydata.org/pandas-docs/version/0.17.0/generated/pandas.DataFrame.iteritems.html#pandas.DataFrame.iteritems)). However `flake8-bugbear` flags uses of this function:

```
$ cat example.py
import pandas as pd

df = pd.DataFrame({"A": [1, 2, 3], "B": [4, 5, 6]})
print(list(df.iteritems()))
$ flake8 example.py
example.py:4:12: B301 Python 3 does not include `.iter*` methods on dictionaries. Remove the `iter` prefix from the method name. For Python 2 compatibility, prefer the Python 3 equivalent unless you expect the size of the container to be large or unbounded. Then use `six.iter*` or `future.utils.iter*`.
```

My environment:

```
$ python --version
Python 3.6.5
$ flake8 --version
3.5.0 (flake8-bugbear: 18.2.0, mccabe: 0.6.1, pycodestyle: 2.3.1, pyflakes: 1.6.0) CPython 3.6.5 on Darwin
```

**Comment by ambv** [comment]
Thanks for your report!

Flake8 is a primitive linter which doesn't understand which type does your name hold. Most of `iteritems()` is in fact invalid. In fact, even in Pandas, `iteritems()` is just an alias over the `items()` method.

What I'm suggesting is either:
* listen to the linter and rename `iteritems()` to `items()` for Pandas usage as well; or
* disable B301 in your .flake8 config.

As much as I would like the warning to be smarter, this is impossible with the view of the world that Flake8 provides.
